src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig

```diff
@@ -0,0 +1,93 @@
+    <strong>Translation</strong>
+</span>
+{% endblock %}
+
+{% block panel %}
+    {% if collector.messages is empty %}
+        <h2>Translations</h2>
+        <p>
+            <em>No translations have been called.</em>
+        </p>
+    {% else %}
+        {{ block('panelContent') }}
+    {% endif %}
+{% endblock %}
+
+{% block panelContent %}
+    <h2>Called Translations</h2>
+    <ul>
+        <li><strong>Defined messages: {{ collector.countdefines }}</strong></li>
+        <li><strong>Fallback messages: {{ collector.countfallbacks }}</strong></li>
+        <li><strong>Missing messages: {{ collector.countmissings }}</strong></li>
+    </ul>
+
+    <table>
+        <tr>
+            <th>State</th>
+            <th>Locale</th>
+            <th>Domain</th>
+            <th>Id</th>
+            <th>Message Preview</th>
+        </tr>
+        {% for message in collector.messages %}
+            <tr>
+                <td><code>{{ translator.state(message) }}</code></td>
+                <td><code>{{ message.locale }}</code></td>
+                <td><code>{{ message.domain }}</code></td>
+                <td><code>{{ message.id }}</code></td>
+                <td><code>{{ message.translation }}</code></td>
```

Comment:
what about translation messages which are quite long ? this happens regularly in apps which are fully translated